outlook/src/taskpane/components/CollapseSection/CollapseSection.tsx

```diff
@@ -10,6 +10,7 @@ type CollapseSectionProps = {
     isCollapsed: boolean;
     onCollapseButtonClick: () => void;
     hasAddButton?: boolean;
+    hasDropdownAddButton?: boolean,
```

Comment:
This is useful to know if the add button of the collapse section has a dropdown, it's only used with project for now, but it can be used for other use cases in the future.